src/TrixiAtmo.jl

```diff
@@ -24,8 +24,9 @@ include("solvers/solvers.jl")
 include("semidiscretization/semidiscretization_hyperbolic_2d_manifold_in_3d.jl")
 
 export CompressibleMoistEulerEquations2D
+export CompressiblePotentialEulerEquations2D
```

Comment:
We should discuss the names a bit. This is different from the moist Euler equations since there are no "potential Euler equations" - it's just using another variable. Maybe something like `CompressibleEulerEquationsPotentialTemperature2D` could be a bit clearer?